libraries-5/src/test/java/com/baeldung/githubapi/RepositoryLiveTest.java

```diff
@@ -0,0 +1,100 @@
+package com.baeldung.githubapi;
+
+import com.google.common.base.Charsets;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class RepositoryLiveTest {
+    private static final Logger LOG = LoggerFactory.getLogger(RepositoryLiveTest.class);
+
+    @Test
+    void whenWeListAUsersRepositories_thenWeCanAccessTheRepositories() throws IOException {
+        GitHub gitHub = GitHub.connectAnonymously();
+
+        GHUser user = gitHub.getUser("eugenp");
+        List<GHRepository> repositoriesList = user.listRepositories().toList();
+        assertTrue(repositoriesList.size() > 0);
```

Comment:
We have the convention/guideline to use AssertJ for assertions like this, e.g.`assertThat(..).isNotEmpty()`